ops/services/okta-app/main.tf

```diff
@@ -1,5 +1,6 @@
 locals {
-  is_prod = var.env == "prod"
+  is_prod    = var.env == "prod"
+  is_preview = (var.env == "dev" || var.env == "test" || var.env == "pentest")
```

Comment:
I don't think we need this, since the resource it's being used for shouldn't be using a `count` - we should have an instance per environment.

Reply:
We don't need this, but not because every env having a trusted origin, per your previous comment. I'll move this discussion to the resource.